lms/djangoapps/verify_student/models.py

```diff
@@ -8,7 +8,7 @@
 `SoftwareSecurePhotoVerification`. The hope is to keep as much of the
 photo verification process as generic as possible.
 """
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
```

Comment:
Are we sure this is safe, rather than checking each string individually?

Reply:
I reviewed the file and I think we're okay.  Since django models work in unicode, the expectation from reading the docs and looking at examples is that this should be a safe change.

Also Django's porting guide suggests this approach.